Description: pass big-endian online regression tests
 An attempt to fix upstream ticket #2396 and considered experimental
 for Debian.
Origin: http://trac.osgeo.org/postgis/changeset/11979
Author: Paul Ramsey <pramsey@opengeo.org>
Last-Update: 2013-09-24

--- a/regress/tickets.sql
+++ b/regress/tickets.sql
@@ -364,7 +364,7 @@
 
 -- #690 --
 SELECT '#690';
-SELECT ST_MakeLine(ST_GeomFromText('POINT(-11.1111111 40)'), ST_GeomFromText('LINESTRING(-11.1111111 70,70 -11.1111111)')) As result;
+SELECT encode(ST_AsEWKB(ST_MakeLine(ST_GeomFromText('POINT(-11.1111111 40)'), ST_GeomFromText('LINESTRING(-11.1111111 70,70 -11.1111111)')),'ndr'), 'hex') As result;
 
 -- #693 --
 SELECT '#693a', encode(ST_AsEWKB(ST_GeomFromEWKT('SRID=4326;POLYGONM((-71.1319 42.2503 1,-71.132 42.2502 3,-71.1323 42.2504 -2,-71.1322 42.2505 1,-71.1319 42.2503 0))'),'ndr'),'hex');
@@ -790,8 +790,8 @@
 -- #2035 END --------------------------------------------------------------
 
 
--- #2084 --
-SELECT '#2048', num, ST_Within('POINT(-54.394 56.522)', "the_geom"), ST_CoveredBy('POINT(-54.394 56.522)', "the_geom")
+-- #2048 --
+SELECT '#2048', num, ST_Within('POINT(-54.394 56.522)', the_geom), ST_CoveredBy('POINT(-54.394 56.522)', the_geom)
 FROM ( VALUES
 (1, '0103000000010000000E00000051C6F7C5A5324BC02EB69F8CF13F4C40F12EA4C343364BC0326AA2CF47434C402BC1A8A44E364BC02A50E10852434C407F2990D959364BC0A0D1730B5D434C404102452C62364BC0ECF335CB65434C400903232F6B364BC0F635E84B6F434C40BD0CC51D6F364BC0D2805EB873434C40B9E6E26F7B364BC0F20B93A982434C40D9FAAF73D3344BC0FE84D04197444C40BD5C8AABCA344BC0CED05CA791444C4023F2237EC5344BC02A84F23E8E444C40BDCDD8077B324BC0C60FB90F01434C409FD1702E65324BC04EF1915C17404C4051C6F7C5A5324BC02EB69F8CF13F4C40'::geometry), 
 (2, '0103000000010000001C00000003F25650F73B4BC098477F523E3E4C40C9A6A344CE3C4BC0C69698653E3E4C40BDD0E979373E4BC0081FA0FB723E4C400FD252793B3E4BC01A137F14753E4C40537170E998414BC070D3BCE314414C4023FC51D499474BC0D4D100DE024F4C40638C47A984454BC024130D52F0504C40B9442DCDAD404BC03A29E96168554C40C7108DEE20404BC07C7C26FBE7554C40195D6BEF533F4BC0E20391459A564C40239FE40E9B344BC08C1ADB6B41514C40132D3F7095314BC0BA2ADF33124F4C409DB91457952D4BC02C7B681F2B4C4C4089DC60A8C32C4BC07C5C3810924B4C40D7ED409DF22A4BC0F64389963C4A4C405D1EF818AC2A4BC00EC84274084A4C401B48A46DFC294BC0B271A8DF85494C40E78AA6B393294BC01ED0EFFB37494C4081C64B3789294BC0DC5BE7DF2E494C409B23329287294BC0F0D6974E2D494C40CD22D5D687294BC0844316D72C494C40F5229D4FE2294BC002F19825AB484C40A3D0BD5AE9294BC06C0776A9A2484C409FD1702E65324BC04EF1915C17404C409F860AA7BD324BC0162CA390E33F4C40539A5C1C23334BC0FE86B04EB03F4C4081511DFF90334BC088FF36D4873F4C4003F25650F73B4BC098477F523E3E4C40'::geometry), 
--- a/regress/tickets_expected
+++ b/regress/tickets_expected
@@ -114,7 +114,7 @@
 #687|f
 #689|f
 #690
-010200000003000000F771D98DE33826C00000000000004440F771D98DE33826C000000000008051400000000000805140F771D98DE33826C0
+010200000003000000f771d98de33826c00000000000004440f771d98de33826c000000000008051400000000000805140f771d98de33826c0
 #693a|0103000060e61000000100000005000000ea95b20c71c851c02b1895d409204540000000000000f03f9cc420b072c851c0c7bab88d062045400000000000000840b1506b9a77c851c08e75711b0d20454000000000000000c0ff21fdf675c851c0f2d24d6210204540000000000000f03fea95b20c71c851c02b1895d4092045400000000000000000
 #693b|0103000060e61000000100000007000000ea95b20c71c851c0aa605452272045400000000000000000386744696fc851c04703780b2420454000000000000034408638d6c56dc851c04703780b2420454000000000000034c08638d6c56dc851c0e3a59bc42020454000000000000014408638d6c56dc851c08048bf7d1d20454000000000000010409cc420b072c851c04703780b242045400000000000001840ea95b20c71c851c0aa605452272045400000000000003e40
 #694
